pkgs/tools/misc/esphome/default.nix

```diff
@@ -49,6 +49,6 @@ in python.pkgs.buildPythonApplication rec {
     description = "Make creating custom firmwares for ESP32/ESP8266 super easy";
     homepage = "https://esphome.io/";
     license = licenses.mit;
-    maintainers = with maintainers; [ dotlambda globin elseym ];
+    maintainers = with maintainers; [ globin elseym ];
```

Comment:
How about you? @makefu

Reply:
Are you still interested in maintaining esphome @globin?